tests/testutils/test_lint_module_output_update.py

```diff
@@ -6,15 +6,20 @@
 
 from __future__ import annotations
 
+import shutil
 from collections.abc import Callable
 from pathlib import Path
 
 import pytest
 
-from pylint.constants import PY38_PLUS
-from pylint.testutils import FunctionalTestFile
+from pylint.constants import IS_PYPY, PY38_PLUS, PY39_PLUS
+from pylint.testutils import FunctionalTestFile, LintModuleTest
 from pylint.testutils.functional import LintModuleOutputUpdate
 
+FIXTURE_DIRECTORY = Path(__file__).parent / "data/functional"
+DIRECTORIES = list(FIXTURE_DIRECTORY.iterdir())
```

Comment:
Let's not make this a constant. It increases the start up time for every test invocation.

Reply:
We then need to compute it twice to have the proper ids ?

Reply:
It would use the full path that is very long though.

Reply:
The ids are only necessary for `pytest -k ID` right? Then a long name isn't that much of an issue.

Reply:
0ac141cd8